Assets/Scripts/ScriptableObjects/WeaponsSystem/WeaponShootStrategies/IWeaponShootStrategy.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MonoBehaviours.WeaponsSystem;
```

Comment:
I think that some of the reload times need to be fine tuned, but that can be worked on during the playtesting phase